etl/requitements.txt

```diff
@@ -0,0 +1,13 @@
+certifi==2020.6.20
```

Comment:
Please include the requirements in the root folder.

Reply:
Will do. Thanks for letting me know